public/application.js

```diff
@@ -19,10 +17,144 @@ $(document).ready(function(){
       },
       dataType: 'JSON',
       success: function(response){
+        console.log("Create Account",response);
+      }
+    });
+
+    $('#signUpSubmit').click(function(){
+      event.preventDefault();
+      console.log('hello signup');
+      var email = $('.signUp > input[id="email"]');
+      var username = $('.signUp > input[id="username"]');
+      var password = $('.signUp > input[id="password"]');
+
+      account.signIn(username,password);
+    });
+  };
+
+  Account.prototype.signIn = function(username,password){
+    $.ajax({
+      context: this,
+      type: 'POST',
+      url: '/sessions',
+      data: {
+        user: {
+          username: username.val(),
+          password: password.val()
+        }
+      },
+      dataType: 'JSON',
+      success: function(response){
+        console.log("Account logged in",response);
+      }
+    });
+  };
+  Account.prototype.signOut = function(){
+    $.ajax({
+      type: 'DELETE',
+      url: '/sessions',
+      dataType: 'JSON',
+      success: function(response){
+        console.log(response);
+        // this.checkLogin();
+      },
+      error: function(response) {
         console.log(response);
       }
     });
+  };
+
+  var account = new Account();
+
+  $('#signUpSubmit').click(function(){
+    event.preventDefault();
+    console.log('hello signup');
+    var email = $('.signUp > input[id="email"]');
+    var username = $('.signUp > input[id="username"]');
+    var password = $('.signUp > input[id="password"]');
+
+    account.signUp(username,email,password);
+  });
+
+  $('#signInSubmit').click(function(){
+    event.preventDefault();
+    console.log('hello signin');
+    var username = $('.signIn > input[id="siusername"]');
+    var password = $('.signIn > input[id="sipassword"]');
+
+    account.signIn(username,password);
+  });
+
+  $('#signOut').click(function(){
+    event.preventDefault();
+    console.log('hello signout');
+
+    account.signOut();
   });
+  // $('#signUpSubmit').click(function(){
```

Comment:
Intention is to get user's input once user fill signup info and clicks the
button "signUpSubmit"

On 6 August 2015 at 10:37, Fer Martin <notifications@github.com> wrote:

> In public/application.js
> <https://github.com/WDI-HK-8/allaboutgrowth/pull/8#discussion_r36377006>:
>
> >    });
> > +  // $('#signUpSubmit').click(function(){
>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WDI-HK-8/allaboutgrowth/pull/8/files#r36377006>.
>